libs/common-const/src/gnosis_chain/hack.ts

```diff
@@ -2,22 +2,19 @@
 // + logic for chainId switch to/from xDAI
 import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
-import { XDAI_SYMBOL } from '../tokens'
+import { NATIVE_CURRENCIES, WRAPPED_NATIVE_CURRENCIES } from '../nativeAndWrappedTokens'
 
 const CURRENCY_SYMBOLS_ETH = { native: 'Ether', wrapped: 'WETH' }
 
-const ChainCurrencySymbolsMap: Record<SupportedChainId, { native: string; wrapped: string }> = {
-  [SupportedChainId.MAINNET]: CURRENCY_SYMBOLS_ETH,
-  [SupportedChainId.BASE]: CURRENCY_SYMBOLS_ETH,
-  [SupportedChainId.ARBITRUM_ONE]: CURRENCY_SYMBOLS_ETH,
-  [SupportedChainId.GNOSIS_CHAIN]: { native: XDAI_SYMBOL, wrapped: 'wxDAI' },
-  [SupportedChainId.POLYGON]: { native: 'POL', wrapped: 'WPOL' },
-  [SupportedChainId.AVALANCHE]: { native: 'AVAX', wrapped: 'WAVAX' },
-  [SupportedChainId.SEPOLIA]: CURRENCY_SYMBOLS_ETH,
-}
-
 export function getChainCurrencySymbols(chainId?: SupportedChainId): { native: string; wrapped: string } {
-  if (!chainId) return ChainCurrencySymbolsMap[SupportedChainId.MAINNET]
+  if (!chainId) return CURRENCY_SYMBOLS_ETH
```

Comment:
Nitpick: `getChainCurrencySymbols` function is used only once in `wrapUnwrapCallback()` and chainId is not falsy there. We could make it not optional